src/utils/useDrag.ts

```diff
@@ -63,10 +63,19 @@ export const useDrag = ({
       },
       onDrag: ({ event }) => {
         // Compute normalized mouse coordinates (screen space)
-        const nx =
-          ((event.clientX - (clientRect?.left ?? 0)) / size.width) * 2 - 1;
-        const ny =
-          -((event.clientY - (clientRect?.top ?? 0)) / size.height) * 2 + 1;
+        let clientX, clientY;
+        if (event instanceof MouseEvent || event instanceof PointerEvent) {
+          clientX = event.clientX;
+          clientY = event.clientY;
+        } else if (event instanceof TouchEvent) {
+          clientX = event.touches[0].clientX;
+          clientY = event.touches[0].clientY;
+        } else {
+          return;
+        }
+
+        const nx = ((clientX - (clientRect?.left ?? 0)) / size.width) * 2 - 1;
+        const ny = -((clientY - (clientRect?.top ?? 0)) / size.height) * 2 + 1;
```

Comment:
Hmmm playing around with this and the types for `event` seem wrong. It's hitting the `else { return }` block every time

This should work with no ts complaints tho

```
      onDrag: ({ xy }) => {
        // Compute normalized mouse coordinates (screen space)
        const nx = ((xy[0] - (clientRect?.left ?? 0)) / size.width) * 2 - 1;
        const ny = -((xy[1] - (clientRect?.top ?? 0)) / size.height) * 2 + 1;
```

Reply:
Will do tomorrow, thanks!🫡